webapp/lib/charts.dart

```diff
@@ -322,6 +323,26 @@ class DriverTimeseriesBarChartView {
       ..classes.add('anchor')
       ..text = 'Filter Metrics');
     chartContainer.insertAdjacentElement('afterbegin', metricsSelector);
+
+    yUpperLimitRangeSlider = new DivElement()
+      ..classes.add('range-slider-container')
+      ..append(new DivElement()..classes.add('range-value'))
+      ..append(
+        new RangeInputElement()
+        ..value = '0'
+        ..step = '1'
+        ..min = '0'
+        ..onInput.listen((e) {
+          var slider = (e.currentTarget as RangeInputElement);
+          var sliderIndicator = (e.currentTarget as Element).previousElementSibling;
+          var newValue = (int.parse(slider.value) - int.parse(slider.min)) * 100 / (int.parse(slider.max) - int.parse(slider.min));
+          var newPosition = 150 - (newValue * 2.15);
+          sliderIndicator.children.clear();
+          sliderIndicator.append(new Element.span()..text = slider.value);
+          sliderIndicator.style.setProperty('top', 'calc(${-newValue}% + (${newPosition}px))');
```

Comment:
same as below - I'd prefer it if we could do this without using `calc`, can you have a quick look to see if that's possible?

Reply:
As discussed earlier, I'll create an issue to address this.